lib/http/request.js

```diff
@@ -104,6 +105,22 @@ class HttpRequest extends EventEmitter {
     return this.httpResponse.res.statusCode;
   }
 
+  isRedacted(data, reqOptions) {
+    if (reqOptions?.method !== 'POST'){
+      return false;
+    }
+    if (!reqOptions?.path.endsWith('/value')){
+      return false;
+    }
+    for (const value of Object.values(data)) {
+      if (typeof(value) === 'string' && value.startsWith(Key.NULL)) {
+        return true;
+      }
+    }
+
+    return false;
+  }
+
```

Comment:
Alternatively, instead of doing this or even having a `this.redact` property in this file, just create an `else if` block [after this](https://github.com/nightwatchjs/nightwatch/blob/ae23267a194e0d9041904ed340536b069669625e/lib/http/request.js#L247-L256) and if `method` is `POST` and path ends with `/value`, replace the `params.text` with `****` and `params.value` with `'****'.split('')`.

I think that would be a more cleaner solution here and will also look much better in the final report.

Reply:
Agreed, it's much cleaner that way. Updated in b06c329